src/client/styles/Grid.ts

```diff
@@ -5,60 +5,94 @@ import { from } from '@guardian/src-foundations/mq';
 
 export enum COLUMNS {
   MOBILE = 4,
+  MOBILE_MEDIUM = 4,
+  MOBILE_LANDSCAPE = 4,
   TABLET = 12,
   DESKTOP = 12,
+  LEFT_COL = 14,
   WIDE = 16,
 }
 
 enum COLUMN_WIDTH {
   MOBILE = 'minmax(0, 1fr)',
+  MOBILE_MEDIUM = 'minmax(0, 1fr)',
+  MOBILE_LANDSCAPE = 'minmax(0, 1fr)',
   TABLET = '40px',
   DESKTOP = '60px',
+  LEFT_COL = '60px',
   WIDE = '60px',
 }
 
 enum SPACING {
-  MOBILE = space[3],
+  MOBILE = 10,
+  MOBILE_MEDIUM = 10,
```

Comment:
weird that there's no value in the source library for this. Is it worth adding it?

Reply:
It's because 10px isn't a standard source unit. Source units are normally in multiples of 4(px).

After speaking to the source team yesterday, they're going to be reviewing the spacing with the designer to allow for non standard units like this, as this isn't the only place where non-standard units are used.